### PR TITLE
Expose getDeviceSize to the API

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @AlexanderMoskovkin @georgiy-abbasov @VasilyStrelyaev 
The determination of viewport sizes must be performed by `testcafe` to enable this functionality in all browser providers. So there is no need for second form of `resize` API.

### CODE REVIEW

File: src/api/get-device-size.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { readSync } from 'read-file-relative';
+
+
+const DEVICE_DATABASE = JSON.parse(readSync('../../data/devices.json'));
+
+/** Gets the size of the viewport for the specified device.
+ * @function
+ * @name getDeviceSize
+ * @param {string} deviceName - Specifies the name of the target device. You can use the values specified in the Device Name column of [this table]{@link http://viewportsizes.com/}.
+ */
```
Comment (VasilyStrelyaev) [inline src/api/get-device-size.js]:
```
/** Gets the viewport size for the specified device.
 * @function
 * @name getViewportSize
 * @param {string} deviceName - Specifies the name of the target device. Use values from the Device Name column of [this table]{@link http://viewportsizes.com/}.
 */
```

**Comment by AndreyBelym** [comment]
Failed to fetch `devices.json` on build.

**Comment by VasilyStrelyaev** [comment]
\r-

File: src/api/get-device-size.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+export default function (deviceName) {
+    deviceName = deviceName.toLowerCase().split(' ').join('');
```
Comment (georgiy-abbasov) [inline src/api/get-device-size.js]: Maybe trim this way - ` deviceName.toLowerCase().replace(/\s+/g, ''); ` ?

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by inikulin** [comment]
I guess `update-devices-database` has failed.

**Comment by inikulin** [comment]
Maybe remove `update-devices-database` from build dependencies?

**Comment by AndreyBelym** [comment]
FPR

**Comment by georgiy-abbasov** [comment]
lgtm

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "testcafe-browser-natives",
-  "version": "0.11.0",
+  "version": "0.12.0",
```
Comment (AlexanderMoskovkin) [inline package.json]: This is breaking change, so we should update the `major` number. The removed `resize` function is used in [TestCafe](https://github.com/DevExpress/testcafe/blob/master/src/test-run/index.js#L326)

Comment (AlexanderMoskovkin) [inline package.json]: though, 0.11.0 -> 0.12.0 it's a major changing, so lgtm

**Pull request merged**